portal-web/test/functional/com/liferay/portalweb/macros/BlogsEntry.macro

```diff
@@ -420,6 +420,28 @@ definition {
 			entryContentEdit = "${entryContentEdit}");
 	}
 
+	macro editEntryInlineImageURL {
+		if (isSet(uploadFileName)) {
```

Comment:
@Jayda-4145 I don't think we need the `if{...}` block here, since nothing within the new macro is outside the if-block.